javascript/javascript1/week4/main.js

```diff
@@ -0,0 +1,97 @@
+// Voice Assistant
+
+
+let personName = "";
+ const todoList = [];
+
+ function getReply(command) {
+    if (command === "" || typeof command !== "string") {
+        alert("Give some command Please!");
+    }
+    else if (command.includes("Hello my name is") && personName == "") {
+        personName = command.split(" ").pop();
+        return `Nice to meet you ${personName}`;
+
+    } else if (command.includes("Hello my name is") && personName != "") {
+        return `${personName} Hi again!`;
+
+    } else if (command === "what is my name?") {
+        return `Your name is ${personName}`;
+
+     } if (command === "Add fishing to my todo") {
+         todoList.push("fishing");
+         return todoList[0] + "added to your todoList.";
+
+     } else if (command === "Add singing in the shower to my todo") {
+         todoList.push("Singing");
+         return todoList[1] + "added to your todoList.";
+
+     } else if (command === "What is on my todo?") {
+         return "You have " + todoList.length + " in list to do ' " + todoList + " '";
+
+     } else if (command === "Remove fishing from my todo") {
+         const indexOfFishing = todoList.splice(todoList.indexOf("fishing"), 1);
+         return "Removed " + indexOfFishing + " from your todoList."
```

Comment:
It would be great if you can write a function that would check which todo is written in the command and it would remove that particular todo from the `todoList`. You would call the function in all cases where the command is `Remove ____ from my todo.` You should also check if the todo is in the `todoList`.